<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterChainEditorWindow</class>
 <widget class="QDialog" name="FilterChainEditorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1073</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="chainEditorTitleLabel">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Lyne Filter Chain Editor</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="filterChainPanel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>351</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>Current Chain</string>
   </property>
   <widget class="QListWidget" name="filterChainListWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>58</y>
      <width>211</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="downButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>208</y>
      <width>28</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="autoRaise">
     <bool>false</bool>
    </property>
    <property name="arrowType">
     <enum>Qt::DownArrow</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="upButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>178</y>
      <width>28</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="autoRaise">
     <bool>false</bool>
    </property>
    <property name="arrowType">
     <enum>Qt::UpArrow</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="editFilterButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>68</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>edit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="toggleEnableDisableFilterButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>98</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>toggle enable</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeFilterButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>128</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>remove</string>
    </property>
   </widget>
   <widget class="QComboBox" name="filterTypeComboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>28</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="addFilterButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>28</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>add</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="videoFormatGroupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>251</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Video Format</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>221</width>
      <height>116</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="videoFormatLayout">
     <item>
      <layout class="QHBoxLayout" name="durationLineLayout">
       <item>
        <widget class="QLabel" name="durationLabel">
         <property name="text">
          <string>Duration (sec)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="durationLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="fpsLineLayout">
       <item>
        <widget class="QLabel" name="fpsLabel">
         <property name="text">
          <string>FPS                 </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="fpsLineEdit">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="xPixelsLineLayout">
       <item>
        <widget class="QLabel" name="xPixelsLabel">
         <property name="text">
          <string>X-Pixels</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="xPixelsLineEdit">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="yPixelsLineLayout">
       <item>
        <widget class="QLabel" name="yPixelsLabel">
         <property name="text">
          <string>Y-Pixels</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="yPixelsLineEdit">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="businessNameLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>37</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Business Name</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="grayScaleCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Gray Scale</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="invertCheckBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Invert</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="chainEditorPlayerWindow">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>100</y>
     <width>591</width>
     <height>441</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Background-color: rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>550</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveSetupButton">
   <property name="geometry">
    <rect>
     <x>691</x>
     <y>550</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Setup</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startTunerButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>550</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Tuner</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
